enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SelectEmbedOptionsStep.tsx

```diff
@@ -66,6 +67,14 @@ export const SelectEmbedOptionsStep = () => {
               }
             />
           )}
+
+          {isBrowserEmbed && (
+            <Checkbox
+              label={t`Allow editing dashboards and questions`}
+              checked={!settings.readOnly}
+              onChange={(e) => updateSettings({ readOnly: !e.target.checked })}
```

Comment:
Other settings in the options step all begins with the word "Allow", so I negate the condition so that it's consistent with other settings.